sponge/src/main/java/com/vexsoftware/votifier/sponge/VotifierPlugin.java

```diff
@@ -208,8 +250,7 @@ public void onServerStart(GameStartedServerEvent event) {
 
         serverGroup = new NioEventLoopGroup(1);
 
-        new ServerBootstrap()
-                .channel(NioServerSocketChannel.class)
+        new ServerBootstrap().channel(NioServerSocketChannel.class)
```

Comment:
I believe that was an accidental change, I was first trying to go about it a different way.  I haven't worked with bukkit/spigot API in a while and I have not worked with bungee plugins at all.  Since everything is already setup on my end I wouldn't mind taking a look and adding it, I don't have servers for those right now to test my changes though.